PCA_Comp_Analysis_Dredging.R

```diff
@@ -510,30 +636,111 @@ write.csv(df, "C:/Users/ksanf/Documents/SD/DFs/dry_PC4.csv", row.names=FALSE)
 #############################################################################
 ###make dredge tables  ##do this 1 pc at a time
 #make model with all remaining predictors after eliminating based on multicollinearity
+
+##########################################################################
+# OET: This section has to be done by the user who builds the model
+# Advantage of using specific variable names
+# + pre-selection can be done by users
+# + adjustment for each PC mode possible
+# + transparent and physical meaning attached with names of predictors
+# Cons:
+# - not a plug-and-play script !
+# - modification must be done every time when one predictor is changed
+##########################################################################
+
+#<OET 2020-12-17>
+# res<-glm(y~omega + merid_transport700 + hgt1000 + merid_trans925 + zonal_trans_925 + zonal_trans_700  + pottmp1000_850 + air2m, df, family=gaussian, na.action = "na.fail")
+# #wet season pc 2
+# res<-glm(y~omega +merid_transport700 + airdiff +merid_trans925 +zonal_trans_925 + hgt500 + zonal_trans_700 + pottmp1000_850 + skt +pr_wtr + shum925 + shum700, df, family=gaussian, na.action = "na.fail")
+# #wet season pc 3
+# res<-glm(y~ omega + merid_transport700 + airdiff + zonal_trans_925 + hgt500 + zonal_trans_700 + pottmp1000_850 + air2m + shum925 + shum700, df, family=gaussian, na.action = "na.fail")
+# #wet season pc4
+# res<-glm(y~omega+ merid_transport700 + +hgt1000+ airdiff+ merid_trans925 + zonal_trans_925 + hgt500 + zonal_trans_700 + pottmp1000_850 + air2m + pr_wtr+ shum925 +shum700, df, family=gaussian, na.action = "na.fail")

```

Comment:
are lines 652 through 658 repeated below? can these be removed?

Reply:
Yes you can remove the commented lines, I kept them so we can check that the predictors are correct in the modified code lines. Note, you can include variable  _airdiff_ again in your branch (I don't have the data file for _airdiff_.